document-models/budget-statement/src/tests/scenario.test.ts

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable @typescript-eslint/no-unsafe-call */
```

Comment:
commented out because the `document-model-specs` dir no longer exists and trying to scan for the dir breaks the code